finalPrep/modZeroHero.js

```diff
@@ -1,49 +1,73 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName = "McKendy";
+var specialAbility = "Passing all tests";
 
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
+var greeting = "Well hello there";
+var catchphrase = "Stay up";
 
 // Declare two variables - power AND energy - set to integers
+var power = 9000;
+var energy = 9001;
 
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
+var fullPower = (power * 500);
+var fullEnergy = (energy + 150);
 
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = false;
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ["McRobs A. Lot", "Mr. Mean Man", "Bane"]
+var sidekick = ["Ms. Helps A. Lot", "Aunt Jemima", "Classic Robin"]
 // Print the first sidekick to your console
-
+console.log(sidekick [0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2]);
 // Write some code to add a new archEnemy to the archEnemies array
-
+archEnemies.push("Mid Life Crisis Man");
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemies);
 // Remove the first sidekick from the sidekicks array
-
+sidekick.shift();
 // Print the sidekicks array to console to ensure you added a new sidekick
-
+console.log(sidekick);
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
+function assessSituation(dangerLevel, saveTheDay, badExcuse){
+  console.log(dangerLevel, saveTheDay, badExcuse)
+if (dangerLevel >= 50) {
+  console.log(badExcuse);
+} else if (dangerLevel < 50) {
+  console.log(saveTheDay);
+}else if (dangerLevel <= 9.9) {
```

Comment:
what if the `dangerLevel == 9.99` ? Can you come up with a better way to code this conditional?